sat/parser.py

```diff
@@ -81,7 +81,6 @@ def parse_args(self, args=None, namespace=None):
             return parsed
 
     def parse_known_args(self, args=None, namespace=None):
-        self.exit_on_error = False
         return super().parse_known_args(args=args, namespace=namespace)
```

Comment:
It looks like this method is now no different than the version from the parent class, `ArgumentParser`. Would it be equivalent if we just removed this method definition entirely now?

Reply:
Looks like it, I'll go ahead and do that instead, thanks.